commands/initial/templates/protractor.conf.js

```diff
@@ -0,0 +1,39 @@
+const {SpecReporter} = require('jasmine-spec-reporter');
+const WebpackDevServer = require('webpack-dev-server');
+const webpack = require('webpack');
+const config = require('./webpack/webpack.dev');
+
+const PORT = 9001;
+
+exports.config = {
+    allScriptsTimeout: 5000,
+    specs: [
+        './e2e/**/*.e2e-spec.ts'
+    ],
+    capabilities: {
+        'browserName': 'chrome'
```

Comment:
I know I'm being picky, but these quotes aren't needed, right? If not, I like to keep attribute names unquoted unless necessary.